core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/logprovider/factory.go

```diff
@@ -52,6 +52,11 @@ const (
 	BufferVersionV1      BufferVersion = "v1"
 )
 
+var (
+	// RPC nodes can provide logs as far back as 128 blocks
+	lookbackBlocks = int64(128)
```

Comment:
can we add a bit of buffer, since it will take some time to process these logs. maybe 100?